apis/python/src/tiledb/vector_search/index.py

```diff
@@ -354,9 +354,21 @@ def open_updates_array(self, timestamp: int = None):
                 timestamp = int(time.time() * 1000)
             return tiledb.open(self.updates_array_uri, mode="w", timestamp=timestamp)
 
-    def consolidate_updates(self, **kwargs):
+    def consolidate_updates(
+        self, 
+        reuse_centroids: bool = True,
```

Comment:
nit: `reuse_centroids` -> `retrain_index`

Centroids is only relevant to the IVF index. Training can be relevant for other indexing algos.